zerver/tests/test_users.py

```diff
@@ -306,6 +306,15 @@ def test_create_user_backend(self) -> None:
         result = self.client_post("/json/users", valid_params)
         self.assert_json_error(result, "Disposable email addresses are not allowed in this organization")
 
+        # Don't allow user to sign up with email aliases based on + when realm
```

Comment:
This should say "Don't allow creating a user with + in their email address when ..."